content/docs/command-reference/params/index.md

```diff
@@ -145,51 +146,38 @@ $ dvc run -n train -d logs/ -o users.csv \
 
 ## Examples: Python parameters file
 
-Consider this parameters file in Python format, named `params.py`:
+Consider this Python parameters file named `params.py`:
 
 ```python
-IS_BOOL: bool = True
-CONST = 5
-
-# All standard variable types are supported
-FLOAT = 0.001
+BOOL = True
+INT = 5
 STR = 'abc'
-DICT = {
-    "a": 1,
-    "b": 2
-}
-LIST = [1, 2, 3]
```

Comment:
what is the reason removing this?

Reply:
Why do we have a full list of Python types? We already mentioned that "All standard variable types are supported". Probably users won't be learning Python on our site. Only `BOOL` and `INT` are used later in the example so I left those and a couple more to make the point.

> As mentioned in https://github.com/iterative/dvc.org/pull/1799#discussion_r504869926

Reply:
TBH it's not even ideal to have these notes inside the Python code example IMO but OK, we probably don't want to make a whole new section in the description above to explain all the rules around Python params files.

Reply:
I think it was very explicit and easy to see all the posibilities at once, when you move something into text you make it harder to grasp the idea ... it's not a strong preference, tbh, I think it was good enough we spent some time on it, it's not a very core functionality

Reply:
No strong opinion either. Reverted this here for now.